huaweicloud/services/fgs/data_source_huaweicloud_fgs_function_events.go

```diff
@@ -0,0 +1,127 @@
+// Generated by PMS #238
+package fgs
+
+import (
+	"context"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/go-uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/tidwall/gjson"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/helper/httphelper"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/helper/schemas"
+)
+
+func DataSourceFunctionEvents() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceFunctionEventsRead,
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: `The region in which to query the resource.`,
+			},
+			"function_urn": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `The function URN to which the events belong.`,
+			},
+			"events": {
+				Type:        schema.TypeList,
+				Computed:    true,
+				Description: `All events that match the filter parameters.`,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"id": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: `The event ID.`,
+						},
+						"name": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: `The event name.`,
+						},
+						"updated_at": {
```

Comment:
This field in resource `huaweicloud_fgs_function_event` is string value.

Reply:
OFC, already uses the RFC3339 string to replace it